cmd/terramate/cli/cli_init_test.go

```diff
@@ -207,6 +208,7 @@ func TestInit(t *testing.T) {
 	} {
 		t.Run(tc.name, func(t *testing.T) {
 			s := sandbox.New(t)
+
```

Comment:
This empty line is leftover of other changes or added to improve readability?

Reply:
Im kinda stupid with adding newlines because it looks better to me, but I would not generalize that, can remove :laughing:

Reply:
Or in this case it could have been a leftover, anyway I tried to improve it now